### PR TITLE
PL: PD deep dive workshop map

### PR DESCRIPTION
**Pull request by breville**
https://code.org/professional-development-workshops already has a map showing CSF workshops, retrieved from the server using https://studio.code.org/dashboardapi/v1/pd/k5workshops.

Now, this map shows only intro workshops, rather than intro and deep-dive workshops, though later in 2019 the plan is have it show both again.

Also now, https://code.org/professional-development-deep-dive gets a version of that map which shows only deep-dive CSF workshops, retrieved from the server using https://studio.code.org/dashboardapi/v1/pd/k5workshops?deep_dive_only=1:

Screenshot 2019-05-15 16 14 17

Also, the workshop edit page (at something like https://studio.code.org/pd/workshop_dashboard/workshops/116/edit) now shows the "Should this appear on the K-5 workshop map?" option even for deep-dive workshops.

Screenshot 2019-05-15 16 08 27

### CODE REVIEW

**Review by islemaster — APPROVED** [review]
:tada:

**Review by agealy — CHANGES_REQUESTED** [review]
Let's hold off on releasing this until we resolve how to deal with deep dive workshops showing up on the new map and the original CSF map

File: apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx [hunk 1]
```diff
@@ -434,8 +434,7 @@ export class WorkshopForm extends React.Component {
```
Comment (Erin007) [inline apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx:434]: For another PR on another day, can we camelCase these jsx and js file names?

**Review by Erin007 — APPROVED** [review]
Code LGTM, but happy to take another look if needed when @agealy's note is addressed - just let me know.

**Review by agealy — APPROVED** [review]

**Review by Erin007 — APPROVED** [review]

**Pull request merged**